src/components/tooltip/Tooltip.tsx

```diff
@@ -120,13 +148,17 @@ const Tooltip: React.FC<TooltipProps> = ({
             id={id}
             role="tooltip"
             ref={setPopperElement}
-            className={ type === 'tooltip' ? 'tk-tooltip' : 'tk-hint' }
+            className={`${type === 'tooltip' ? 'tk-tooltip' : 'tk-hint'} ${
```

Comment:
You can use the 'classname' lib ;) (https://github.com/JedWatson/classnames#readme)
```
import classnames from 'classnames';
...
className={`classname('tk-tooltip': tooltip, 'tk-hint': !tooltip, className)} 
```